<tt>
When&nbsp;attempting&nbsp;to&nbsp;connect&nbsp;to&nbsp;a&nbsp;MSSQL&nbsp;instance,&nbsp;we&nbsp;see&nbsp;this&nbsp;error:&lt;div&gt;&lt;br&gt;&lt;/div&gt;&lt;div&gt;&lt;div&gt;&amp;lt;pre&amp;gt;System.NullReferenceException:&nbsp;Object&nbsp;reference&nbsp;not&nbsp;set&nbsp;to&nbsp;an&nbsp;instance&nbsp;of&nbsp;an&nbsp;object&lt;/div&gt;&lt;div&gt; &nbsp;at&nbsp;Mono.Data.Tds.Protocol.TdsConnectionPool.GetConnection&nbsp;()&nbsp;[0x00000]&nbsp;in&nbsp;&amp;amp;lt;filename&nbsp;unknown&amp;amp;gt;:0 &lt;/div&gt;<br>
&lt;div&gt; &nbsp;at&nbsp;System.Data.SqlClient.SqlConnection.Open&nbsp;()&nbsp;[0x00000]&nbsp;in&nbsp;&amp;amp;lt;filename&nbsp;unknown&amp;amp;gt;:0 &lt;/div&gt;&lt;div&gt; &nbsp;at&nbsp;SubSonic.SqlDataProvider.CreateConnection&nbsp;(System.String&nbsp;newConnectionString)&nbsp;[0x00000]&nbsp;in&nbsp;&amp;amp;lt;filename&nbsp;unknown&amp;amp;gt;:0 &lt;/div&gt;<br>
&lt;div&gt; &nbsp;at&nbsp;SubSonic.SqlDataProvider.CreateConnection&nbsp;()&nbsp;[0x00000]&nbsp;in&nbsp;&amp;amp;lt;filename&nbsp;unknown&amp;amp;gt;:0 &lt;/div&gt;&lt;div&gt; &nbsp;at&nbsp;SubSonic.AutomaticConnectionScope..ctor&nbsp;(SubSonic.DataProvider&nbsp;provider)&nbsp;[0x00000]&nbsp;in&nbsp;&amp;amp;lt;filename&nbsp;unknown&amp;amp;gt;:0 &lt;/div&gt;<br>
&lt;div&gt; &nbsp;at&nbsp;SubSonic.SqlDataProvider.GetReader&nbsp;(SubSonic.QueryCommand&nbsp;qry)&nbsp;[0x00000]&nbsp;in&nbsp;&amp;amp;lt;filename&nbsp;unknown&amp;amp;gt;:0 &lt;/div&gt;&lt;div&gt; &nbsp;at&nbsp;SubSonic.DataService.GetReader&nbsp;(SubSonic.QueryCommand&nbsp;cmd)&nbsp;[0x00000]&nbsp;in&nbsp;&amp;amp;lt;filename&nbsp;unknown&amp;amp;gt;:0 &lt;/div&gt;<br>
&lt;div&gt; &nbsp;at&nbsp;SubSonic.Query.ExecuteReader&nbsp;()&nbsp;[0x00000]&nbsp;in&nbsp;&amp;amp;lt;filename&nbsp;unknown&amp;amp;gt;:0 &lt;/div&gt;&lt;div&gt; &nbsp;at&nbsp;SubSonic.ReadOnlyRecord`1[Adzerk.Model.Network].LoadByParam&nbsp;(System.String&nbsp;columnName,&nbsp;System.Object&nbsp;paramValue)&nbsp;[0x00000]&nbsp;in&nbsp;&amp;amp;lt;filename&nbsp;unknown&amp;amp;gt;:0 &lt;/div&gt;<br>
&lt;div&gt; &amp;lt;...&amp;gt;&lt;/div&gt;&lt;div&gt; &nbsp;at&nbsp;System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodFilter&nbsp;(IActionFilter&nbsp;filter,&nbsp;System.Web.Mvc.ActionExecutingContext&nbsp;preContext,&nbsp;System.Func`1&nbsp;continuation)&nbsp;[0x00000]&nbsp;in&nbsp;&amp;amp;lt;filename&nbsp;unknown&amp;amp;gt;:0 &lt;/div&gt;<br>
&lt;div&gt; &nbsp;at&nbsp;System.Web.Mvc.ControllerActionInvoker+&amp;amp;lt;&amp;amp;gt;c__DisplayClassd+&amp;amp;lt;&amp;amp;gt;c__DisplayClassf.&amp;amp;lt;InvokeActionMethodWithFilters&amp;amp;gt;b__c&nbsp;()&nbsp;[0x00000]&nbsp;in&nbsp;&amp;amp;lt;filename&nbsp;unknown&amp;amp;gt;:0 &lt;/div&gt;<br>
&lt;div&gt; &nbsp;at&nbsp;System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodWithFilters&nbsp;(System.Web.Mvc.ControllerContext&nbsp;controllerContext,&nbsp;IList`1&nbsp;filters,&nbsp;System.Web.Mvc.ActionDescriptor&nbsp;actionDescriptor,&nbsp;IDictionary`2&nbsp;parameters)&nbsp;[0x00000]&nbsp;in&nbsp;&amp;amp;lt;filename&nbsp;unknown&amp;amp;gt;:0 &lt;/div&gt;<br>
&lt;div&gt; &nbsp;at&nbsp;System.Web.Mvc.ControllerActionInvoker.InvokeAction&nbsp;(System.Web.Mvc.ControllerContext&nbsp;controllerContext,&nbsp;System.String&nbsp;actionName)&nbsp;[0x00000]&nbsp;in&nbsp;&amp;amp;lt;filename&nbsp;unknown&amp;amp;gt;:0&nbsp;&amp;lt;/pre&amp;gt;&lt;/div&gt;&lt;/div&gt;&lt;div&gt;&lt;br&gt;<br>
&lt;/div&gt;&lt;div&gt;Does&nbsp;anyone&nbsp;know&nbsp;what&nbsp;causes&nbsp;this?&nbsp;We&amp;#39;re&nbsp;able&nbsp;to&nbsp;connect&nbsp;to&nbsp;the&nbsp;MSSQL&nbsp;server&nbsp;from&nbsp;other&nbsp;machines&nbsp;in&nbsp;the&nbsp;network.&nbsp;This&nbsp;is&nbsp;the&nbsp;only&nbsp;instance&nbsp;running&nbsp;Mono&nbsp;(on&nbsp;Ubuntu).&lt;/div&gt;&lt;div&gt;&lt;br&gt;&lt;/div&gt;&lt;div&gt;Thanks,&lt;/div&gt;&lt;div&gt;<br>
&lt;br&gt;&lt;/div&gt;&lt;div&gt;James&lt;/div&gt;&lt;div&gt;&lt;br&gt;&lt;/div&gt;<br>

</tt>
